### PR TITLE
feat: show birthdates

### PR DESCRIPTION
**Pull request by tbirrell**
A quick addition to show birthdates I made to my instance some time ago. I saw it was requested in #2358 and figured I'd make a PR. 

This is what it looks like

screenshot-monica-2019 01 29-12-59-11

### CODE REVIEW

File: resources/views/people/_header.blade.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
           @if ($contact->birthdate && !($contact->is_dead))
             @if ($contact->birthdate->getAge())
               <span class="{{ htmldir() == 'ltr' ? 'mr1' : 'ml1' }}">@include('partials.icons.header_birthday')</span>
-              <span>{{ $contact->birthdate->getAge() }}</span>
+              <span>{{$contact->birthdate->toShortString()}} ({{ $contact->birthdate->getAge() }})</span>
```
Comment (asbiin) [inline resources/views/people/_header.blade.php]: Don't use Carbon::toShortString but DateHelper::getShortDate instead:
```suggestion
              <span>{{ \App\Helpers\DateHelper::getShortDate($contact->birthdate) }} ({{ $contact->birthdate->getAge() }})</span>
```

Comment (asbiin) [reply resources/views/people/_header.blade.php]: My bad, I didn't saw birthdate is a SpecialDate, so it' ok ;)

**Comment by djaiss** [comment]
That's a good idea. Thanks for making and submitting this PR.

**Comment by Venhaus** [comment]
Thank you! Small but very helpful change.

**Pull request merged**